charts/ssi-credential-issuer/values.yaml

```diff
@@ -88,11 +88,16 @@ migrations:
       memory: 200M
   seeding:
     testDataEnvironments: ""
```

Comment:
should be removed because consortia files are getting removed

Reply:
removed